src/components/20-molecules/list/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@axa-ch/list",
+  "version": "0.0.0-beta.0",
+  "description": "The list component for the AXA Pattern Library",
+  "author": "Pattern Warriors",
+  "homepage": "https://github.com/axa-ch/patterns-library/tree/develop/src/components/20-molecules/list#readme",
+  "license": "Copyright 2019 AXA Versicherungen AG",
```

Comment:
I've waited for this moment 😁

Reply:
Maybe remove 2019 from create-amo script

Reply:
I will ask Luca if that is somehow needed of if he just wrote it without any obligation.